tests/examples/pacer/nef/s3/cob.json

```diff
@@ -0,0 +1,26 @@
+{
+  "appellate": false,
+  "contains_attachments": false,
```

Comment:
Also, I just noticed that in this example `contains_attachments` should be `True`

Since the emails contains:
`Document description: Main Document `
and
`Document description: Proposed/Unsigned Order  `

Reply:
[_contains_attachments_plain](https://github.com/freelawproject/juriscraper/blob/main/juriscraper/pacer/email.py#L365) has a bug, it will always return False
`associated_documents = +1` is not adding 1 per each doc seen, but redefining the variable as the number `+1`
I am correcting this in the PR